pkg/resources/management.cattle.io/v3/globalrole/validator.go

```diff
@@ -126,14 +127,28 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 	}
 	err = auth.ConfirmNoEscalation(request, clusterRules, "", a.grbResolver)
 	if err != nil {
-		return admission.ResponseFailedEscalation(err.Error()), nil
+		hasEscalate, escErr := auth.RequestUserHasVerb(request, gvr, a.sar, escalateVerb, newGR.Name, "")
```

Comment:
We can't gaurantee that we made the request (i.e. if you aren't trying to escalate clusterRules but are trying to escalate in globalRules we still need to make the request later on). I can try to do something to cache the result, but it will make the code more complex.

My thinking was the extra network call (being that it only happens in cases where you are trying to escalate in both buckets) is rare enough to justfiy keeping the code as-is for quality reasons. But I can look at it again if you think the savings are worth it.

Reply:
I changed to immediately return if you are ever detected to have bind permissions. Resolving this thread, that should solve the concerns around the extra network call.